src/common/gui/base/buttons/OutlineButton.ts

```diff
@@ -0,0 +1,65 @@
+import m, { Children, ClassComponent, Vnode } from "mithril"
+import { lang, TranslationKey } from "../../../misc/LanguageViewModel.js"
+import { ClickHandler } from "../GuiUtils.js"
+import { BaseButton, BaseButtonAttrs } from "./BaseButton.js"
+import { theme } from "../../theme.js"
+
+export interface OutlineButtonAttrs {
+	label: TranslationKey
+	text?: TranslationKey
+	click?: ClickHandler
+	disabled?: boolean
+	expanded?: boolean
+}
+
+/**
+ * Simple outline button component
+ * @see Component attributes: {OutlineButtonAttrs}
+ * @example
+ * m(OutlineButton, {
+ * 	   label: button.label,
+ * 	   click: button.click,
+ * 	   disabled: button.isReadOnly,
+ * }),
+ */
+export class OutlineButton implements ClassComponent<OutlineButtonAttrs> {
+	view({ attrs }: Vnode<OutlineButtonAttrs>): Children {
+		return m(BaseButton, {
+			label: lang.getMaybeLazy(attrs.label),
+			text: attrs.text ? lang.getMaybeLazy(attrs.text) : lang.getMaybeLazy(attrs.label),
+			onclick: attrs.click,
+			disabled: attrs.disabled,
+			style: {
+				borderColor: theme.content_message_bg,
+				color: theme.content_button,
+			},
+			class: this.resolveClasses(attrs.expanded, attrs.disabled).join(" "),
```

Comment:
You can call the join inside the function and return just the string, I think it's a bit more clean